src/project-manager.ts

```diff
@@ -672,18 +668,14 @@ export class InMemoryLanguageServiceHost implements ts.LanguageServiceHost {
 	}
 
 	/**
-	 * @param fileName relative or absolute file path
+	 * @param filePath absolute file path
 	 */
-	getScriptSnapshot(fileName: string): ts.IScriptSnapshot | undefined {
-		let exists = this.fs.fileExists(fileName);
-		if (!exists) {
-			fileName = path.posix.join(this.rootPath, fileName);
```

Comment:
I wonder whether there was a reason this code was put in place... Are you positive this can only ever be called with an absolute file path?